scrabble.js

```diff
@@ -1,8 +1,138 @@
-var Scrabble = function() {};
+var Scrabble = function() {
+  this.LETTER_SCORES = [['a','e','i','o','u','l','n','r','s','t'], ['d','g'], ['b','c','m','p'], ['f','h','v','w','y'], ['k'], [], [], ['j','x'], [], ['q','z']];
+};
+
+Scrabble.prototype.score = function(string) {
+  var word = string.toLowerCase();
+  var wordScore = 0;
+  // Loop through each letter of the word
+  for (k = 0; k < word.length; k++) {
```

Comment:
By not having the `var` in front of this and the `j` variable below, you are accidentally creating global variables. This will definitely cause issues later if you try to use the same iteration variable.